compiler/cpp/Makefile.am

```diff
@@ -100,9 +100,8 @@ thrift_SOURCES += src/thrift/generate/t_as3_generator.cc \
                   src/thrift/generate/t_lua_generator.cc \
                   src/thrift/generate/t_netcore_generator.cc \
                   src/thrift/generate/t_netcore_generator.h \
-                  src/thrift/generate/t_ocaml_generator.cc \
-                  src/thrift/generate/t_perl_generator.cc \
```

Comment:
Looks like you removed perl, ocaml, and php...

Reply:
Ooopsie ...